packages/flutter/test/widgets/text_test.dart

```diff
@@ -1662,29 +1662,48 @@ void main() {
   });
 
   testWidgets('Mouse hovering over selectable Text uses SystemMouseCursor.text', (WidgetTester tester) async {
-    await tester.pumpWidget(const MaterialApp(
-      home: SelectionArea(
-        child: Text('Flutter'),
+    await tester.pumpWidget(
```

Comment:
Is the original test no longer valid? otherwise, we should create a new test

Reply:
yes, the original test is no longer valid